### ISSUE TITLE
Landing Page v 1.0

### ISSUE THREAD

**Issue by cynepton** [BuildForSDGCohort2/Team-085-Frontend#6]
## This will be the first iteration of the Landing page.
React must be used to build the landing page, use Class components whether or not there will be states

### Features
**The landing page must contain** *(You can use see [fiverr's site](https://www.fiverr.com/) if you need a guide)*
- A **search bar**: that takes user data and can sends it to a back-end server for processing. Dummy data should be used for now, but when the server endpoint is exposed, the URL should simply be added to the front-end to make it work. Preferably, an AJAX fetch request should be implemented. 

- **Buttons**: The landing page should contain buttons for:
  - Sign-in and Log-in, which would link to [Auth0](https://auth0.com/), which will be used as our authentication platform. For now, leave the link field empty.

More features will be added to this issue until we have a working version of our landing page. The landing page should not be too integrated as it would be a separate service when we eventually refactor to a microservices architecture

**Comment by mayowa-ola** [comment]
Alright...Will get to work
You don't think we should use react hooks as functional components rather than class components for managing our state?

**Comment by mayowa-ola** [comment]
I guess I can start work right? and which branch are we to push to?

**Comment by cynepton** [comment]
Yeah, I'm not exactly done setting up yet, but information on development is available in the [CONTRIBUTING.md](https://github.com/BuildForSDGCohort2/Team-085-Frontend/blob/develop/CONTRIBUTING.md), and information on setting up the development environment is available in the [README.md](https://github.com/BuildForSDGCohort2/Team-085-Frontend/blob/develop/CONTRIBUTING.md)

I would alert you on the slack channel once you can start development. In the meantime you could go through the features so you can get your wireframe ready

**Comment by cynepton** [comment]
> Alright...Will get to work
> You don't think we should use react hooks as functional components rather than class components for managing our state?

I haven't used hooks, but if you're cool with using them you can go ahead just make sure to comment your code clearly so no one gets confused

**Comment by mayowa-ola** [comment]
Alright...Noted

**Comment by cynepton** [comment]
@mayowa-ola How is it coming up?

Remember not to do too much in a single Pull Request, so that it can be easily reviewed. A couple components are enough for a single PR. One error in a single PR will cause the entire PR to be rejected.

**Comment by mayowa-ola** [comment]
Alright...will do a PR by tomorrow

**Comment by mayowa-ola** [comment]
Hi...I can't push to dev to do a PR.
Kindly check
Annotation 2020-08-29 191303](https://user-images.githubusercontent.com/26276075/91643506-b470b500-ea2b-11ea-91f0-9bf48439e2f6.png)

**Comment by cynepton** [comment]
The reason you're having this issue is because you didn't follow the instructions in the CONTRIBUTING.md file. In the contribution section, i stated how to make correct pull requests. See link [here](https://github.com/BuildForSDGCohort2/Team-085-Frontend/blob/develop/CONTRIBUTING.md#how-to-contribute) 

Take your time to go through the entire document and the README.md.

You have gotten the entire process wrong. You cloned the upstream repository directly. You are supposed to first
- fork this repository
- write your code on your dev branch
- push your code to your staging branch to test locally. 
- Then make a PR from your staging branch which will be your `origin`, to the main repository which will be the `upstream`.

**Comment by cynepton** [comment]
Go through the necessary process, take your time, perfection is more valuable than speed.

**Comment by mayowa-ola** [comment]
Well noted...I have created the PR...Please review
